build-livecd

```diff
@@ -37,11 +46,15 @@ done
 cat >> $KS_INSTALL <<KS
 %post
 chmod +x \
+  /etc/dhcp/dhclient.d/get-zip-server.sh \
   /etc/rc.d/rc.local \
-  /usr/bin/discover-host \
-  /usr/bin/find-missing-libs \
-  /usr/bin/find-largest-rpms \
+  /usr/bin/discovery-autostart \
   /usr/bin/discovery-debug \
+  /usr/bin/discovery-register \
+  /usr/bin/discovery-setup \
+  /usr/bin/find-largest-rpms \
+  /usr/bin/find-missing-libs \
+  /usr/bin/get-zip-server_config
   /usr/bin/nm-prepare
 %end
 KS
```

Comment:
This is strange, I've upgraded to Fedora 21 and ksflatten now fails requiring me to specify `-c` option. It looks like this option was there for years. Can you please add it? So instead of

    ksflatten "$1" -o $KS_TARGET

do

    ksflatten -c "$1" -o $KS_TARGET

and check it works for you as well. Thanks.